client/src/App.js

```diff
@@ -19,6 +20,8 @@ import AdminLogin from "./components/auth/AdminLogin";
 import { AdminEditProfile } from "./components/dashboard/AdminEditProfile";
 import { AddNewPet } from "./components/dashboard/AddNewPet";
 import { loadAdmin } from "../src/actions/auth";
+import { UserEditProfile } from "./components/dashboard/UserEditProfile";
+import { loadUser } from "../src/actions/auth";
```

Comment:
You can add `loadUser` in line 22 if they are exporting from the same file path: import { loadAdmin, loadUser } from "../src/actions/auth";